core/common/lib/validator-lib/src/test/java/org/eclipse/edc/validator/jsonobject/JsonObjectValidatorTest.java

```diff
@@ -217,6 +218,73 @@ void shouldFail_whenInputIsNull() {
                 });
     }
 
+    @Test
```

Comment:
please wrap these tests specific to the `OptionalIdArray` in a `@Nested` dedicated class

Reply:
done